### PR TITLE
Adding contact information as per #1504

### PR DESCRIPTION
**Pull request by lubianat**

### CODE REVIEW

File: docs/contact_us.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+If you do not have a GitHub account, you can sign up for a free account [here](https://github.com/).
+
+## E-mail and other contacts
+
+The Cell Ontology also has a [Slack Channel](https://anatomy-and-cell-onto.slack.com) and monthly meetings, with an [open agenda](https://docs.google.com/document/d/1XvMbNvr0FEsdqGhg79BYCYEHSqUxRHMcvhbGizEAht8/edit). 
+
+If you want to join the channel or talk about any other topics related to the Cell Ontology, you may reach out to one of the active CL editors. 
+The official contact for CL listed at the OBO Foundry is [Alexander Diehl](mailto:addiehl@buffalo.edu), but feel free to contact any other individual. A list of active CL editors can be found [here](http://obophenotype.github.io/uberon/team/).
+
+Note that requests on the [GitHub issue tracker](https://github.com/obophenotype/cell-ontology/issues/) are might be picked up quicker, as individuals are often busy.
```
Comment (shawntanzk) [inline docs/contact_us.md:15]: typo 'are' should be removed. My preference would also be to nudge further by using "will probably" heh

**Review by shawntanzk — APPROVED** [review]

File: docs/contact_us.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
+# Contact Us
+
+## GitHub
+The preferred point of contact for the Cell Ontology is the GitHub [Issue Tracker](https://github.com/obophenotype/cell-ontology/issues/). It is used commonly for requesting new terms or suggesting changes to existing terms. Feel free to open an issue with any kind of request/question/consideration you might have.
```
Comment (paolaroncaglia) [inline docs/contact_us.md:4]: To make it even easier, I'd suggest replacing
"Feel free to open an issue"
with
"Feel free to open a new issue"
and hyperlinking "open a new issue" with https://github.com/obophenotype/cell-ontology/issues/new/choose.

File: docs/contact_us.md [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+The preferred point of contact for the Cell Ontology is the GitHub [Issue Tracker](https://github.com/obophenotype/cell-ontology/issues/). It is used commonly for requesting new terms or suggesting changes to existing terms. Feel free to open an issue with any kind of request/question/consideration you might have. 
+
+If you do not have a GitHub account, you can sign up for a free account [here](https://github.com/).
+
+## E-mail and other contacts
+
+The Cell Ontology also has a [Slack Channel](https://anatomy-and-cell-onto.slack.com) and monthly meetings, with an [open agenda](https://docs.google.com/document/d/1XvMbNvr0FEsdqGhg79BYCYEHSqUxRHMcvhbGizEAht8/edit). 
+
+If you want to join the channel or talk about any other topics related to the Cell Ontology, you may reach out to one of the active CL editors. 
+The official contact for CL listed at the OBO Foundry is [Alexander Diehl](mailto:addiehl@buffalo.edu), but feel free to contact any other individual. A list of active CL editors can be found [here](http://obophenotype.github.io/uberon/team/).
```
Comment (paolaroncaglia) [inline docs/contact_us.md:13]: So we've resolved not to use the cl-edit Google mailing list? Fine with me, just checking. But please go ahead anyway; the cl-edit contact may be added later if we decide to.

File: docs/contact_us.md [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+
+Note that requests on the [GitHub issue tracker](https://github.com/obophenotype/cell-ontology/issues/) will probably be picked up quicker, as individuals are often busy.
```
Comment (paolaroncaglia) [inline docs/contact_us.md:15]: I'd replace "will probably be picked up quicker" with "may be addressed more quickly".

**Review by paolaroncaglia — COMMENTED** [review]
@lubianat I added a few comments. Thank you.

File: docs/contact_us.md (hunk 3 above)
Comment (lubianat) [reply docs/contact_us.md:13]: @paolaroncaglia I think we have not reached a consensus on that, but I agree, we can add it later :)

**Comment by lubianat** [comment]
@paolaroncaglia is everything okay with the docs?  @shawntanzk can you merge and update the documentation? I am not 100% if it will be updated via GH actions

**Comment by shawntanzk** [comment]
> @shawntanzk can you merge and update the documentation?

Happy to once there's a positive review from @paolaroncaglia 

> I am not 100% if it will be updated via GH actions

It is not automatically triggered atm, so you need to go into actions and trigger it manually. In case you want to do it in the future and you arent sure where it is: 
Screenshot 2022-03-31 at 13 41 07

**Review by paolaroncaglia — APPROVED** [review]

**Comment by paolaroncaglia** [comment]
@lubianat @shawntanzk all good with me! Thank you both.

**Pull request merged**